### PR TITLE
Added solution for preorder traversal in Java

### PR DESCRIPTION
**Pull request by SubhamPramanik**
Added solution for 

https://www.hackerrank.com/challenges/tree-preorder-traversal/problem

in Java

### CODE REVIEW

File: Data Structures/Trees/PostOrderTraversal_Solution.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+*/
+
+public static void postOrder(Node root) {
+
+        if (root != null) {
+            postOrder(root.left);
+            postOrder(root.right);
+            System.out.print(root.data + " ");
+        }
+
```
Comment (Kvaibhav01) [inline Data Structures/Trees/PostOrderTraversal_Solution.java]: Remove extra line spaces in lines 30 and 36.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Rename path from _Data Structures/Trees/PostOrderTraversal_Solution.java_ to _Data Structures/Trees/**Tree: Postorder Traversal**/Solution.java_.

Also do the same in the second file.

**Comment by SubhamPramanik** [comment]
Made the specified changes.

**Review by Kvaibhav01 — APPROVED** [review]
Good work!

**Pull request merged**